### DISCUSSION TITLE
Disable default mappings

### DISCUSSION THREAD

**Discussion by kapral18** [arsham/archer.nvim#1] [Q&A]
can we allow disabling default mappings yet allowing to pass new opts without merging into base mappings.

Would be nice to have `default_mappings = true|false`. What do you think?

**Comment by arsham** [comment]
This is a nice idea. I have made an [issue](https://github.com/arsham/archer.nvim/issues/2) for this. I am a bit busy these days, but I will try to fix this issue soon. You are welcome to make a PR if you like.

**Reply by arsham** [reply]
@kapral18 the `default_mappings` boolean has been added to the config. Thank you for the recommendation.
